docs/feature-versioning.json

```diff
@@ -0,0 +1,40 @@
+{
+  "/docs/agents/agent-approval": "1.0.0",
```

Comment:
We might need a regex instead of a string as index, so we can match all versions.

But for now, this is not needed.

Same for the version string, I don't use it in code right now, but later we could use the `semver` package to reliable check if something is/was available in a certain version.